src/gui/src/scriptWidget.cpp

```diff
@@ -232,7 +253,7 @@ void ScriptWidget::addTextToOutput(const QString& text, const QColor& color)
 void ScriptWidget::readSettings(QSettings* settings)
 {
   settings->beginGroup(objectName());
-  input_->readSettings(settings);
+  // input_->readSettings(settings);
```

Comment:
Why?

Reply:
because input is created once the init is called for either python or tcl so it's a nullptr, I haven't figured out the best method to solve this yet.